test/tests/multiapps/loose_couple_time_adapt/adaptiveDT.i

```diff
@@ -0,0 +1,60 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  xmin = 0
+  xmax = 1
+  ymin = 0
+  ymax = 1
+[]
+
+[Variables]
+  active = 'u'
```

Comment:
Get rid of the active lines, they are not needed. And in the other input files as well.